.github/workflows/testing.yml

```diff
@@ -26,7 +26,8 @@ jobs:
         run: pip install -e .
       - name: Run pytest with coverage
         run: pytest --cov=./ --cov-report xml:./nwbinspector/nwbinspector/coverage.xml
-      - name: Upload coverage to Codecov
+      - if: ${{ matrix.python-version == '3.9' && matrix.os == 'ubuntu-latest' }}
```

Comment:
Also added this line to constrain unnecessary Codecov uploads, at least until we make Python version-specific or platform-specific checks (not sure we ever will, though...)